framework/src/modules/interoperability/types.ts

```diff
@@ -159,6 +161,7 @@ export interface CCCommandExecuteContext {
 	networkIdentifier: Buffer;
 	eventQueue: EventQueue;
 	ccm: CCMsg;
-	getAPIContext: () => ImmutableAPIContext;
+	getAPIContext: () => APIContext;
```

Comment:
only interoperability module will use CCAPIContext, and API will never use CCAPIContext or call it. The user when executes a ccm, then it can expect to have api to other modules to process something so its needed here